core/templates/pages/library-page/search-bar/search-bar.component.css

```diff
@@ -184,8 +181,8 @@ oppia-search-bar .oppia-input-group.classroom-page-input-group {
     padding-top: 8px;
   }
   oppia-search-bar .oppia-input-group input {
-    border-bottom-left-radius: 10px;
-    border-top-left-radius: 10px;
+    border-bottom-left-radius: 10px !important;
+    border-top-left-radius: 10px !important;
```

Comment:
Please try to achieve the intended behaviour without employing the use of `!important`. [See here for why](https://uxengineer.com/css-specificity-avoid-important-css/) (TL;DR - Managing stylesheets filled with the `!important` property is extremely hard, we should instead try to achieve the required layout/style by, for example, varying the specificity of the selectors used).

Reply:
hi @nithinrdy, I have made the changes by removing important and checking specificity.

Reply:
@nithinrdy PTAL